agent/src/main/java/io/atomix/agent/AtomixAgent.java

```diff
@@ -165,7 +165,7 @@ public static void main(String[] args) throws Exception {
     }
   }
 
-  static String[] parseAddress(String address) {
+  static String[] parseInfo(String address) {
     String[] parsed = address.split(":");
     if (parsed.length > 3) {
       throw new IllegalArgumentException("Malformed address " + address);
```

Comment:
Trim strings after split?

Reply:
If there's whitespace in the string then it won't be parsed by the command line parser as a single argument and will thus be malformed anyways